pkg/bgpv1/agent/controller.go

```diff
@@ -478,3 +481,16 @@ func (c *Controller) validatePolicy(policy *v2alpha1api.CiliumBGPPeeringPolicy)
 	}
 	return nil
 }
+
+// setDefaultPeerPort sets the PeerPort of the provided neighbor
+// to defaultPeerPort (179) if unset.
+func setDefaultPeerPort(n *v2alpha1api.CiliumBGPNeighbor) {
+	if n.PeerPort == nil {
+		n.PeerPort = PtrPort(types.DefaultPeerPort)
+	}
+}
+
+// PtrPort returns the provided port as a pointer.
+func PtrPort(port uint16) *uint16 {
```

Comment:
Just a question as I am not 100% sure here: wouldn't it be better to use `*uint32` here? 2 reasons:
- [API Conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types) doc is saying "All public integer fields MUST use the Go (u)int32 or Go (u)int64 types"
- For (u)int32 or Go (u)int64 types you could use helpers from [k8s.io/utils/pointer](https://pkg.go.dev/k8s.io/utils/pointer) instead of defining this custom func

Reply:
@rastislavs thanks for catching this. I've updated the PR based on your feedback, PTAL.

Reply:
@rastislavs GH is back online, PTAL when you have a moment.

Reply:
Thanks! LGTM now